docs/getting-started/env-configuration.md

```diff
@@ -785,7 +785,14 @@ modeling files for reranking.
 
 - Type: `int`
 - Default: `3`
-- Description: Sets the default number of results to consider when using RAG.
+- Description: Sets the default number of results to consider for the embedding when using RAG.
```

Comment:
Maybe change to something like this?:
Sets the default number of results to consider from the embeddings vectordb when using RAG.